### ISSUE TITLE
feat: improve create UI

### ISSUE THREAD

**Issue by holkeb** [hypercerts-org/hypercerts#297]

**Comment by holkeb** [comment]
Thanks for the updated UI @ccerv1.

- Let's do "Hypercert Name" or "Hypercert Title"... both are sufficient I think. I'll implement "Hypercert Name"
- "Hypercert Dimensions" and "Advanced Dimensions": I think we confuse people with the word dimensions, so I just used "Fields"... so we have "General Fields", "Hypercert Fields", "Advanced Fields"

I'll update those two things directly in plasmic, when I also add an intro.

Question reg. the allowlist: Can pilot users change the allowlist?

**Comment by holkeb** [comment]
Checked some other minting forms (OpenSea, Rarible) and they don't provide any intro. So, I changed my mind in favor of the simplistic form without an intro. People ending up at the form should already know what creating a hypercert means.

I push the updates on the names. I also changed the form title to "Create New Hypercert" (adding "new")

So, it's only the question reg. the allowlist that's open for me
